lib/recorder.js

```diff
@@ -193,12 +193,12 @@ function generateRequestAndResponse(req, bodyChunks, options, res, dataChunks) {
 //  to finish (which is the point of the test).
 let currentRecordingId = 0
 
-function record(rec_options) {
+function record(options) {
```

Comment:
There are a lot of other things called `options` in this file, and it most often in nock refers to the `options` passed to `http.request`. I don't feel super strongly, though think `recOptions` or `recorderOptions` might be a better name here.